src/types/ModuleUiDialogProps.ts

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2023 Mikhail Aheichyk
+Copyright 2023 Nordeck IT + Consulting GmbH.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/**
+ * Module UI Dialog props.
+ */
+export interface ModuleUiDialogProps {
+    /**
+     * Initial value to enable or disable the dialog submit button.
+     */
+    canSubmit?: boolean;
+
+    /**
+     * Title of the dialog.
+     */
+    title: string;
+
+    /**
+     * Cancel button label.
+     */
+    cancelLabel?: string;
+
+    /**
+     * Action button label.
+     */
+    actionLabel?: string;
+}
```

Comment:
For these optional settings: please explain in the doc-comment what happens if they are undefined.